---
permalink: /
title: "About me"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---
I work on understanding gene architecture evolution and its roles in driving both the molecular diversity and complexity of nervous systems, as well as the breakdown of these systems in disease. Leveraging my formal training in molecular genetics and genomics and neurobiology, I develop and use computational methods for comparative and functional genomics in classical and emerging model systems including octopus and squid, nematodes, fish, and human cell fate reprogramming.

# Experience

| Years        | Experience                                             |
| ------------ | ------------------------------------------------------ |
| 2018-Present | Postdoc -- Stanford University -- advised by Andy Fire |
| 2012-2018    | PhD -- WashU in St. Louis -- advised by Andrew Yoo     |
| 2005-2011    | BA Biology, minor in Chemistry -- University of Utah   |

# Fellowships

| Years     | Fellowship                                   |
| --------- | -------------------------------------------- |
| 2023      | Whitman Early Career Award, MBL              |
| 2021-2022 | Wu Tsai Interdisciplinary Scholar, Stanford	 |
| 2019      | Whitman Early Career Award, MBL              |
| 2018-2019 | Stanford Genome Training Grant, Stanford     |
| 2018      | Grass Fellowship in Neuroscience, MBL        |
| 2014-2015 | IPNG Training Grant, WashU                   |

# Publications
1. Herbert A, Lee D, McCoy MJ, Wucherpfennig J, Behrens V, Kingsley D. Genetic mechanisms of axial patterning in Apeltes quadracus. _Evolution Letters_ (2024). (In Press)
2. McCoy MJ and Fire AZ. Parallel gene size and isoform expansion of ancient neuronal genes. _Current Biology_ (2024), S0960982224001635. DOI: [https://doi.org/10.1016/j.cub.2024.02.021](https://doi.org/10.1016/j.cub.2024.02.021). ([Moroz Dispatch](https://doi.org/10.1016/j.cub.2024.03.016))([Stanford Wu Tsai Neurosciences Institute News](https://neuroscience.stanford.edu/news/neuronal-and-synaptic-genes-expanded-size-and-diversity-during-evolution))([PreLights Review](https://prelights.biologists.com/highlights/ancient-origins-of-complex-neuronal-genes/)). 
3. Baden T, Briseño J, Coffing G, Cohen-Bodénès S, Courtney A, Dickerson D, Dölen G, Fiorito G, Gestal C, Gustafson T, Heath-Heckman E, Hua Q, Imperadore P, Kimbara R, Król M, Lajbner Z, Lichilín N, Macchi F, McCoy MJ, Nishiguchi M K, Nyholm S V, Otjacques E, Pérez-Ferrer P A, Ponte G, Pungor J R, Rogers T F, Rosenthal J J C, Rouressol L, Rubas N, Sanchez G, Pereira C, Schultz D T, Seuntjens E, Songco-Casey J O, Stewart I E, Styfhals R, Tuanapaya S, Vijayan N, Weissenbacher A, Zifcakova L, Schulz G, Weertman W, Simakov O, Albertin C. Cephalopod-omics: emerging fields and technologies in cephalopod molecular biology and evolution. _Biology. Integr. Comp. Biol._ (2023). DOI: [https://doi.org/10.1093/icb/icad087](https://doi.org/10.1093/icb/icad087)
4. Lu YL, Liu Y, McCoy MJ, and Yoo AS. MiR-124 synergism with ELAVL3 enhances target gene
expression to promote neuronal maturity. _Proceedings of the National Academy of Sciences_
118(22):e2015454118 (2021). DOI: [https://doi.org/10.1073/pnas.2015454118](https://doi.org/10.1073/pnas.2015454118)
5. Cates K\*, McCoy MJ\*, Kwon JS\*, Liu Y\*, Abernathy DG, Zhang B, Liu S, Gontarz P, Kim WK, Chen S, Kong W, Ho JN, Burbach KF, Gabel HW, Morris SA, and Yoo AS. Deconstructing stepwise fate
conversion of human fibroblasts to neurons by MicroRNAs. _Cell Stem Cell_ 28, 127-140, e9 (2020). DOI: [https://doi.org/10.1016/j.stem.2020.08.015](https://doi.org/10.1016/j.stem.2020.08.015) *Co-first author. ([Herdy et al. Dispatch](https://doi.org/10.1016/j.stem.2020.12.007))
6. McCoy MJ and Fire AZ. Intron and gene size expansion during nervous system evolution. BMC Genomics
21, 360 (2020). DOI: [https://doi.org/10.1186/s1 2864-020-6760-4](https://doi.org/10.1186/s12864-020-6760-4). ([Stanford Wu Tsai Neuroscience Institute Q&A](https://neuroscience.stanford.edu/news/qa-evolution-octopus-and-squid-brains-could-shed-light-origins-intelligence))
7. Wahba L\*, Jain N\*, Fire AZ\*, Shoura MJ\*, Artiles KL\*, McCoy MJ\* and Jeong DE\*. An Extensive Meta-Metagenomic search identifies SARS-CoV-2-homologous sequences in pangolin lung viromes. _mSphere_ 5(3):e00160-20 (2020). DOI: [https://doi.org/10.1128/mSphere.00160-20](https://doi.org/10.1128/mSphere.00160-20). *All authors contributed equally
and author order was chosen randomly.
8. McCoy MJ\*, Paul AJ\*, Victor MB, Richner M, Gabel HW, Gong H, Yoo AS & Ahn T. LONGO: an R
package for interactive gene length dependent analysis for neuronal identity. _Bioinformatics_ 34.13, i422-i428. (2018). DOI: [https://doi.org/10.1093/bioinformatics/bty243](https://doi.org/10.1093/bioinformatics/bty243). *Co-first author.
9. Abernathy DG\*, Kim W\*, McCoy MJ\*, Lake A, Ouwenga R, Xing X, Li D, Lee HJ, Heuckeroth RO,
Dougherty JD, Wang T, Yoo AS. MicroRNAs induce a permissive chromatin environment that enables
neuronal subtype-specific reprogramming of adult human fibroblasts. _Cell Stem Cell_ 21.3, 332-348 e339 (2017). DOI: [https://doi.org/10.1016/j.stem.2017.08.002](https://doi.org/10.1016/j.stem.2017.08.002). *Co-first author.

# Preprints
1. McCoy MJ and Fire AZ. genesizeR: an R package to incorporate gene size into categorical and
quantitative investigations of gene sets. (Submitted)
2. Herbert AL\*, Allard CAH\*, McCoy MJ\*, Wucherpfennig JI, Krueger SP, Chen HI, Gourlay AN, Jackson
KD, Abbo LA, Bennett SH, Sears JD, Rhyne AL, Bellono NW, Kingsley DM. The genetic basis of novel
trait gain in walking fish. _bioRxiv_ (2023). DOI: [https://doi.org/10.1101/2023.10.14.562356](https://doi.org/10.1101/2023.10.14.562356). *Co-first author. (In Revision)
3. McCoy MJ and Fire AZ. Ancient origins of complex neuronal genes. _bioRxiv_ (2023). DOI: [https://doi.org/10.1101/2023.03.28.534655](https://doi.org/10.1101/2023.03.28.534655). (Now published in _Current Biology_)
4. Wahba L\*, Jain N\*, Fire AZ\*, Shoura M\*, Artiles K\*, McCoy MJ\*, Jeong DE\*. Identification of a pangolin niche for a 2019-nCoV-like coronavirus through an extensive meta-metagenomic search. _bioRxiv_ (2020). DOI: [https://doi.org/10.1101/2020.02.08.939660](https://doi.org/10.1101/2020.02.08.939660). *All authors contributed equally. (Now published in _mSphere_)
